driver.go

```diff
@@ -502,20 +502,36 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	createOpts.ContainerBasicConfig.Labels = driverConfig.Labels
 
 	// Logging
-	switch driverConfig.Logging.Driver {
-	case "", LOG_DRIVER_NOMAD:
-		// Only modify container logging path if LogCollection is not disabled
-		if !d.config.DisableLogCollection {
-			createOpts.LogConfiguration.Driver = "k8s-file"
-
-			createOpts.ContainerBasicConfig.LogConfiguration.Path = cfg.StdoutPath
+	if driverConfig.Logging.Driver != "" {
+		switch driverConfig.Logging.Driver {
+		case LOG_DRIVER_NOMAD:
+			if !d.config.DisableLogCollection {
+				createOpts.LogConfiguration.Driver = "k8s-file"
+				createOpts.ContainerBasicConfig.LogConfiguration.Path = cfg.StdoutPath
+			}
+		case LOG_DRIVER_JOURNALD:
+			createOpts.LogConfiguration.Driver = "journald"
+		default:
+			return nil, nil, fmt.Errorf("Invalid task logging.driver option")
 		}
-	case LOG_DRIVER_JOURNALD:
-		createOpts.LogConfiguration.Driver = "journald"
-	default:
-		return nil, nil, fmt.Errorf("Invalid logging.driver option")
+		createOpts.ContainerBasicConfig.LogConfiguration.Options = driverConfig.Logging.Options
+	} else {
+		d.logger.Trace("no podman log driver provided, defaulting to plugin config")
+		switch d.config.Logging.Driver {
+		case "": // The default is nomad
```

Comment:
Pushed this as the fix for the mentioned "issue", not sure if this should go somewhere else or if it would be possible to always properly initialize the `Logging` struct.

Reply:
Yes, I think this is the correct spot. Just changed this a little in https://github.com/hashicorp/nomad-driver-podman/pull/285/commits/79fa57034c41bbd251915e54ce5a08cfce894828#diff-d16a4232641fc7ed88a3737c22b4f54d7e03dd7fdf9053530fde70e5bca63fb1L521-R521 to avoid the use of `fallthrough` since it can be a little confusing to read.